x/evm/keeper/keeper.go

```diff
@@ -391,3 +397,7 @@ func (k Keeper) AddTransientGasUsed(ctx sdk.Context, gasUsed uint64) (uint64, er
 	k.SetTransientGasUsed(ctx, result)
 	return result, nil
 }
+
+func (k Keeper) GetSdkCtx(handlerId uint64) *sdk.Context {
```

Comment:
```suggestion
func (k Keeper) getSdkCtx(handlerId uint64) *sdk.Context {
```
prefer to use private methods by default

Reply:
Done.